### PR TITLE
Follower: add explanation on roles

### PR DESCRIPTION
**Pull request by docohe**

### CODE REVIEW

**Comment by PRMerger12** [comment]
@docohe : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dorcohe** [comment]
https://github.com/MicrosoftDocs/dataexplorer-docs/issues/119

**Comment by orspod** [comment]
@docohe - is the [manage permissions](#manage-permissions) link no longer relevant?
We need to fix the links so they start with (/azure . I will suggest the changes on your PR.

File: data-explorer/follower.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Attaching a database to a different cluster using the follower capability is use
...
+There are various methods you can use to attach a database. In this article, we discuss attaching a database using C#, Python or an Azure Resource Manager template. To attach a database, you must have user, group, service principal, or managed identity with at least contributor role on the leader cluster and the follower cluster. You can add or remove role assignments using [Azure Portal](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal), [PowerShell](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-powershell), [Azure CLI](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-cli) and [ARM template](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-template). Learn more about [Azure role-based access control (Azure RBAC)](https://docs.microsoft.com/en-us/azure/role-based-access-control/overview) and the [different roles](https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles).
```
Comment (orspod) [inline data-explorer/follower.md:32]:
```suggestion
There are various methods you can use to attach a database. In this article, we discuss attaching a database using C#, Python or an Azure Resource Manager template. To attach a database, you must have user, group, service principal, or managed identity with at least contributor role on the leader cluster and the follower cluster. You can add or remove role assignments using [Azure Portal](/azure/role-based-access-control/role-assignments-portal), [PowerShell](/azure/role-based-access-control/role-assignments-powershell), [Azure CLI](/azure/role-based-access-control/role-assignments-cli) and [ARM template](/azure/role-based-access-control/role-assignments-template). You can learn more about [Azure role-based access control (Azure RBAC)](/azure/role-based-access-control/overview) and the [different roles](/azure/role-based-access-control/rbac-and-directory-admin-roles). 
```

**Comment by PRMerger7** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
#sign-off

**Pull request merged**